Sources/AsyncHTTPClient/ConnectionPool/HTTP1/HTTP1ClientChannelHandler.swift

```diff
@@ -281,7 +281,7 @@ final class HTTP1ClientChannelHandler: ChannelDuplexHandler {
         case .close:
             context.close(promise: nil)
 
-        case .wait:
+        case .wait, .noAction:
```

Comment:
Do we need this new state or can we just re-use `.wait`?

Reply:
reuse `.wait`. `.wait` is a do nothing. can be renamed in a follow up pr.